### PR TITLE
Mobile chat gutter tweak

### PR DESCRIPTION
**Pull request by cecileboucheron**
@adamjspooner

### CODE REVIEW

**Comment by cecileboucheron** [comment]
@adamjspooner bump ^

File: shared/chat/conversation/messages/attachment/image/index.tsx [hunk 1]
```diff
@@ -83,8 +83,13 @@ class ImageAttachment extends React.PureComponent<Props, State> {
         <ShowToastAfterSaving transferState={this.props.transferState} />
         <Kb.Box2 direction="vertical" fullWidth={true}>
           {(!mobileImageFilename || !Styles.isMobile) && (
-            <Kb.Box2 direction="horizontal" fullWidth={true} gap="xtiny">
-              <Kb.Text type="BodyTiny">
+            <Kb.Box2
+              direction="horizontal"
+              fullWidth={true}
+              gap="xtiny"
+              style={{paddingRight: Styles.globalMargins.small}}
```
Comment (adamjspooner) [inline shared/chat/conversation/messages/attachment/image/index.tsx:90]: Let's put this in `styles` below.

File: shared/chat/conversation/messages/wrapper/exploding-meta/index.tsx [hunk 2]
```diff
@@ -230,21 +230,20 @@ const styles = Styles.styleSheetCreate(
           position: 'relative',
         },
         isElectron: {height: 20},
-        isMobile: {height: 24},
+        isMobile: {height: 20},
```
Comment (adamjspooner) [inline shared/chat/conversation/messages/wrapper/exploding-meta/index.tsx:233]: We can put this in `common` now, right?

**Comment by cecileboucheron** [comment]
@adamjspooner updated

**Pull request merged**